packages/wallet-adapter-mui-design/src/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
```

Comment:
nit: this file (`index.tsx`) can only hold an export statement
`export * from "./WalletConnector";`

and create another file, `WalletConnector.tsx` file that holds this current file content.

something simlar to what we have in the antd package
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-ant-design/src/index.tsx